src/main/java/frc/robot/subsystems/superstructure/ShotParameters.java

```diff
@@ -33,11 +33,11 @@ public Parameters interpolate(final Parameters endValue, final double t) {
     );
 
     static {
-        shotDataMap.put(0d, new Parameters(
-                Rotation2d.fromDegrees(0),
-                0,
-                0,
-                0)
+        shotDataMap.put(4d, new Parameters(
+                Rotation2d.fromDegrees(55),
+                80,
+                60,
+                70)
```

Comment:
I think this is our subwoofer shot datapoint, which can stay.

Reply:
Good to be merged but we should test different flywheel velocities later for subwoofer shooting, maybe if we reduced the velocity it would be less likely to bounce out like it did a few times (or the opposite could be true where a higher velocity would help it ricochet into the goal when shooting from the sides of the subwoofer)